src/components/UserActivity/UserActivity.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import {SafeAreaView, View, FlatList, StyleSheet, Text} from 'react-native';
+
+const Item = ({ title, count }) => (
+    <View style={styles.item}>
+        <Text style={styles.title}>{title}</Text>
+        <Text style={styles.count}>{count}</Text>
+    </View>
+);
+
+export default function  UserActivity ({activity}) {
+
+    const renderItem = ( {item} ) => {
+        return  <Item title={item.title} count={item.description}/>
+    }
+    return (
+        <SafeAreaView style={styles.container}>
+            {activity?.items && <FlatList
```

Comment:
What one will see when where is no activity.items?

Reply:
fix